interface/src/app/gene/searchMany/searchMany.js

```diff
@@ -15,37 +15,38 @@ angular.module('adage.gene.searchMany', [
         'main': {
           templateUrl: 'gene/gene-network.tpl.html',
           controller: ['$stateParams', 'MlModelTracker', 'UserFactory',
-          function($stateParams, MlModelTracker, UserFactory) {
```

Comment:
*Note: The following changes (from line 18-49) are simply to correct an indentation issue, which was pointed out in issue #197 , but code did not really change. A couple of documentation lines were changed to stay under the 80-char limit.